wagtail/admin/templates/wagtailadmin/shared/keyboard_shortcuts_dialog.html

```diff
@@ -0,0 +1,58 @@
+{% load wagtailadmin_tags %}
+{% dialog icon_name='regex' id='keyboard-shortcuts-dialog' title=_("Keyboard shortcuts") %}
+    <table class="w-h-full w-overflow-y-scroll">
+        <caption class="w-sr-only">
+            All keyboard shortcuts
+        </caption>
```

Comment:
Just a reminder to use translated values for all text.

Reply:
Ah, sry, but what do you mean by translated values?

Reply:
No problems. Essentially we want to make sure that all strings / text is marked for translation. 

https://docs.wagtail.org/en/stable/contributing/translations.html#marking-strings-for-translation

This leverages a Django system of translation. 

I don't think we need to mark the actual shortcuts for translation but I'll check with the translation team.

Reply:
No need to pass the shortcut keys label (e.g. `Ctrl + z`) through the translation system on this PR. 

In the future we may review how these labels are generated once we start to consider more complex customisation use cases.